test/31_genbranch_caching.bats

```diff
@@ -12,8 +12,8 @@ setup() {
   git clone "$BATS_TEST_TMPDIR/remoterepo" "$BATS_TEST_TMPDIR/testrepo"
   pushd "$BATS_TEST_TMPDIR/testrepo"
   git pile init
-  git config pile.genbranch-user-name = "pile bot"
-  git config pile.genbranch-user-email = "git@pi.le"
+  git config pile.genbranch-user-name "pile bot"
+  git config pile.genbranch-user-email "git@pi.le"
```

Comment:
Maybe this is related to the issues I'm seeing in older versions of git?

Reply:
What issues?

Reply:
nevermind... I was referring to #99, but you already replied there about the config change needed